site/src/components/scroll.md

```diff
@@ -65,3 +65,46 @@ Force an elements overflow to be horizontally scrollable by using `.scroll-verti
 		</pre>
 	</p>
 </div>
+
+## Side scrolling container
+
+Horizontally scroll list of items
```

Comment:
Can we expand on the description a bit more? What's the difference between horizonal scroll and side scroll? When should I use this vs `scroll-horizontal`?

Upon initial look, I was expecting this to be something similar to the behavior of `.scroll-horizontal`, but there was no scroll bar. Then, I clicked on the buttons to see what happens and saw the effect.

Reply:
Agreed on improving description. Need to mention a few things:
1) this is primarily a narrow-view pattern and thus functionality should be tested on mobile
2) this element should likely be hidden or done a different way on larger screens

I actually think it would be preferable if we remove the anchors from this example, unless we're going to update this this PR with the typescript required to make them work well.